.github/workflows/release.yml

```
@@ -0,0 +1,94 @@
+        run: cd apps/desktop && npx electron-builder --linux
+      - name: Upload Linux binaries to release
+        run: |
+          gh release upload ${{ github.event.release.tag_name }} \
+            apps/desktop/out/*.AppImage \
+            apps/desktop/out/*.deb \
+            --clobber
+        env:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+  build-desktop-macos:
+    if: ${{ secrets.CSC_LINK != '' }}
```

actionlint (1.7.12)

[error] context "secrets" is not allowed here. available contexts are "github", "inputs", "needs", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)